[2022-08-27 22:06:08,324] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.model_dag scheduled__2022-08-25T00:00:00+00:00 [queued]>
[2022-08-27 22:06:08,335] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.model_dag scheduled__2022-08-25T00:00:00+00:00 [queued]>
[2022-08-27 22:06:08,335] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-27 22:06:08,336] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-27 22:06:08,336] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-27 22:06:08,353] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): model_dag> on 2022-08-25 00:00:00+00:00
[2022-08-27 22:06:08,360] {standard_task_runner.py:52} INFO - Started process 383 to run task
[2022-08-27 22:06:08,363] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'model_dag', 'scheduled__2022-08-25T00:00:00+00:00', '--job-id', '256', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpy7w2luks', '--error-file', '/tmp/tmpldm9__fc']
[2022-08-27 22:06:08,364] {standard_task_runner.py:80} INFO - Job 256: Subtask model_dag
[2022-08-27 22:06:08,439] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.model_dag scheduled__2022-08-25T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-27 22:06:08,515] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=model_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-25T00:00:00+00:00
[2022-08-27 22:06:49,711] {python.py:173} INFO - Done. Returned value was: None
[2022-08-27 22:06:49,722] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=model_dag, execution_date=20220825T000000, start_date=20220827T220608, end_date=20220827T220649
[2022-08-27 22:06:49,777] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-27 22:06:49,804] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
